.travis.yml

```diff
@@ -16,12 +19,14 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install -g jpm mozilla-download
+  - npm install -g jpm
```

Comment:
Sometime we should probably try to reduce the Travis time generally – there must be tricks, I imagine.  As it is either way we'll just be letting it build and bug us if there's a problem, rather than watching it.